content/en/docs/components/pipelines/v2/cli.md

```diff
@@ -4,119 +4,161 @@ description = "Interact with KFP via the CLI"
 ```
 
-You can get help for a particular command directly on the command line using `--help`:
+### Get help for a command
+
+To get help for a specific command, use the argument `--help` directly in the command line. For example, to view guidance about the `kfp run` command, type the following command:
 
 ```shell
 kfp run --help
 ```
+## Main functons of the KFP CLI
+
+You can use the KFP CLI to do the following:
+
+* [Interact with KFP resources](#cli_interact_with_kfp_resources)
+
+* [Compile pipelines](#cli_compile_pipelines)
 
-The KFP CLI serves three main functions:
-1. Interacting with KFP resources
-2. Compiling pipelines
-3. Building Python-based containerized components
+* [Build containerized Python components](#cli_build_python_components)
 
-## Interact with KFP resources
-The majority of the KFP CLI commands involve creating, reading, updating, or deleting KFP resources via the KFP backend. All of these commands take a common structure:
+### Interact with KFP resources<a id="cli_interact_with_kfp_resources"></a>
+
+The majority of the KFP CLI commands let you create, read, update, or delete KFP resources from the KFP backend. All of these commands use the following general syntax:
 
 ```shell
 kfp <resource_name> <action>
 ```
 
-`<resource_name>` can be one of the following:
+The `<resource_name>` argument can be one of the following:
 * `run`
 * `recurring-run`
 * `pipeline`
 * `experiment`
 
-All resources support the following actions:
+For all values of the `<resource_name>` argument, the `<action>` argument can be one of the following:
 * `create`
 * `list`
 * `get`
 * `delete`
 
-Some resources have resource-specific actions, including:
-* For `run`:
-  * `archive`
-  * `unarchive`
+Some resource names have additional resource-specific actions. The following table lists a few examples of resource-specific actions:
 
-* For `recurring-run`:
-  * `disable`
-  * `enable`
+| Resource name | Additional resource-specific actions
+|---------------|--------
+| `run` | <ul><li>`archive`</li><li>`unarchive`</li></ul>
+| `recurring-run` | <ul><li>`disable`</li><li>`enable`</li></ul>
+| `experiment` | <ul><li>`archive`</li><li>`unarchive`</li></ul>
+| `pipeline` |  | <ul><li>`create-version`</li><li>`list-versions`</li><li>`get-versions`</li><li>`delete-versions`</li></ul>
 
-* For `experiment`
-  * `archive`
-  * `unarchive`
+### Compile pipelines<a id="cli_compile_pipelines"></a>
 
-* For `pipeline`:
-  * `create-version`
-  * `list-versions`
-  * `get-versions`
-  * `delete-versions`
+You can use the `kfp dsl compile` command to compile pipelines or components defined in a Python file to IR YAML.
 
+* To compile a pipeline defined in a Python file, run the following command:
 
-## Compile pipelines
-The KFP supports compiling a pipeline defined in a Python file to YAML using the `kfp dsl compile` command:
+  ```shell
+  kfp dsl compile --py path/to/pipeline.py --output path/to/output.yaml
```

Comment:
Should this bullet take the form as the others: first showing with command placeholders, then showing with command values?